src/redisClient.ts

```diff
@@ -205,6 +248,22 @@ export class RedisClient implements CommonClient {
     return count
   }
 
+  hscanStream(key: string, opt: ScanStreamOptions): ReadableTyped<string[]> {
+    return this.redis().hscanStream(key, opt)
+  }
+
+  async hScanCount(key: string, opt: ScanStreamOptions): Promise<number> {
```

Comment:
should it be in lowercase? (consistent)

```suggestion
  async hscanCount(key: string, opt: ScanStreamOptions): Promise<number> {
```

Reply:
My impression was that you tried to follow the same naming pattern whenever we are just providing a wrapper:
for example `async mget()` wraps `this.redis().mget()` (instead of `async mGet()`).

And you followed camelCase whenever this lib provides extra funtionality, e.g.: `mgetBuffer` or `scanStreamFlat`.

So my logic was:
- `hscanStream()` wraps `this.redis().hscanStream()` call, hence identically named
- `hScanCount()` provides custom functionality (i.e. there is no `redis().hscanCount` call), hence camelCase

TBH I don't know and I don't feel strongly about it, so I trust your judgement in using lowercase